spec/filters/elasticsearch_spec.rb

```diff
@@ -1 +1,13 @@
+# encoding: utf-8
+
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/plugin"
+require "logstash/filters/elasticsearch"
+
+describe LogStash::Filters::Elasticsearch do
+
+  it "should register" do
+    plugin = LogStash::Plugin.lookup("filter", "elasticsearch").new({})
```

Comment:
since the subject is already the class, why not just call subject.register in the assertion? it is an implicit subject, like https://www.relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax